src/pixelator/pna/config/panel.py

```diff
@@ -227,6 +227,29 @@ def _validate_sequences(panel_df, sequence_col):
 
         return errors
 
+    @staticmethod
+    def _validate_marker_names(panel_df):
+        errors = []
+        if any(panel_df.index.str.contains("_")):
+            # Markers should not contain underscores since this messes
+            # things up with Seurat on the R side
+            errors.append(
+                "The marker_id column should not contain underscores. "
+                "Please use dashes instead. Offending values: "
+                f"{panel_df.index[panel_df.index.str.contains('_')]}"
+            )
+        if any(panel_df.index.str.contains("\s")):
+            # Markers should not contain white-spaces since this causes
+            # issues in the demultiplexing step (and other places that
+            # might assume that marker names are single tokens)
+            problematic_lines = panel_df.index[panel_df.index.str.contains("\s")]
+            errors.append(
+                "The marker_id column should not contain white-spaces. "
+                "Please use dashes instead. Offending values: "
```

Comment:
```suggestion
                "Please use dashes instead or remove the white-spaces. Offending values: "
```

Small detail. In the case I had a few weeks ago the white-spaces were leading or trailing so no dashes needed there.